StreamChatSwiftUITests/Tests/ChatChannel/ReactionsUsersView_Tests.swift

```diff
@@ -66,6 +66,6 @@ class ReactionsUsersView_Tests: StreamChatTestCase {
             .frame(width: defaultScreenSize.width, height: 320)
 
         // Then
-        assertSnapshot(matching: view, as: .image(perceptualPrecision: 0.98))
+        assertSnapshot(matching: view, as: .image(perceptualPrecision: 0.97))
```

Comment:
Yeah, this is how it's done on StreamChat and StreamVideo. I just didn't think it was a good idea to implement a snapshot-testing wrapper (as we have in other repos) as part of this PR. Would be great to even share this wrapper across repos, let me create a ticket for this.

Reply:
https://github.com/GetStream/ios-issues-tracking/issues/539

Reply:
even just a constant can help here, since you need to change it in many places, you could add a wrapper afterwards. Sharing it across repos is good idea, we need some test utils which are product agnostic (no chat stuff, no video stuff).

Reply:
Ok, done 👍